cmd2.py

```diff
@@ -2046,6 +2046,9 @@ def select(self, opts, prompt='Your choice? '):
             self.poutput('  %2d. %s\n' % (idx + 1, text))
         while True:
             response = sm.input(prompt)
+            hlen = readline.get_current_history_length()
+            if hlen >= 1 and response != '':
+                readline.remove_history_item(hlen - 1)
```

Comment:
Looks good.  Thanks for this fix.  Not polluting the readline history when using ``select()`` is a nice fix.